### PR TITLE
Community edition: handle special mutation types

### PR DESCRIPTION
**Pull request by cjwooo**
## Description

> Provide description here

## Type of change
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change

## Related issues

> Fix [#1]() 

## Migration notes

> Describe migration notes if any

## Extra info

> Add any additional information

### CODE REVIEW

File: destinations/faros-destination/src/community/hasura-client.ts [hunk 1]
```diff
@@ -132,11 +172,98 @@ export class HasuraClient {
     }
   }
 
+  async writeTimestampedRecord(record: TimestampedRecord): Promise<void> {
+    if (record.operation === Operation.UPSERT) {
```
Comment (tovbinm) [inline destinations/faros-destination/src/community/hasura-client.ts:176]: switch/case would be better here

File: destinations/faros-destination/src/community/hasura-client.ts [hunk 2]
```diff
@@ -132,11 +172,98 @@ export class HasuraClient {
     }
   }
 
+  async writeTimestampedRecord(record: TimestampedRecord): Promise<void> {
```
Comment (tovbinm) [inline destinations/faros-destination/src/community/hasura-client.ts:175]: all these GraphQL operations are invoked one by one (each in separate http request) or in a bulk?

**Review by tovbinm — COMMENTED** [review]
see comments

File: destinations/faros-destination/src/converters/jira/issues.ts [hunk 3]
```diff
@@ -532,7 +531,7 @@ export class JiraIssues extends JiraConverter {
           model: 'tms_TaskReleaseRelationship__Upsert',
           record: {
             at: change.changed.getTime(),
-            where: {
+            data: {
```
[the remark this answers is not in the document]
Comment (cjwooo) [reply destinations/faros-destination/src/converters/jira/issues.ts:534]: This is a bug I'm fixing. It wasn't supposed to be `where`.

[the remark this answers is not in the document]
Comment (tovbinm) [reply destinations/faros-destination/src/converters/jira/issues.ts:534]: Ah, right. It's the `Upsert`!!

**Review by tovbinm — APPROVED** [review]

**Pull request merged**